tools/NuGet/template-service-core/DynamoVisualProgramming.ServiceCoreRuntime.nuspec

```diff
@@ -17,6 +17,8 @@
         </dependencies>
     </metadata>
     <files>
-        <file src="**" target="lib\$TargetFramework$\" exclude="en-US\fallback_docs\*"/>
+              <file src="**"
+              target="lib\$TargetFramework$\"
+              exclude="en-US\fallback_docs\*;NodeHelpSharedDocs\*;Open Source Licenses\*;samples\** "/>
```

Comment:
@aparajit-pratap @pinzart90  do you think it makes sense to invert this and manually include every file we want to deploy so that changes to Dynamo bin folder don't automatically make it to the service?

Reply:
Oh, so this right now will deploy all of the dll's in the bin folder into the service, even those that are not part of the service runtime core?

Reply:
I should have been more clear.
Right now, the way the job is setup for this nuget and build - it will deploy just the `DynamoCore.Sln` - I was just thinking of someone in the future added something to that .sln it will then make into the nuget package automatically.

My worry is that it's not clear what else this DynamoCore.sln might be used for in the future besides the service.

Reply:
Thanks for clarifying. I think this is good enough for now.

Reply:
I think it is easier to control what dlls and content in general within the csproj (more tools at our disposal)
I think we should have a solution (or configs in general) that produce on disk exactly what the service is. It will be valuable to build exactly what will be inside the docker container.

Maybe we should rename the sln to DynamoService.sln (if that is makes it clearer to anyone)

Reply:
`I think we should have a solution (or configs in general) that produce on disk exactly what the service is. It will be valuable to build exactly what will be inside the docker container.` I agree. I will think about if we should change the name, add a config, or add a new .sln.